lib/ide/json-rpc/src/transport/web.rs

```diff
@@ -0,0 +1,2 @@
+//! WebSocket-based `Transport` implementation.
```

Comment:
This file and the reference to this module in `transport.rs` should be removed.
[my fault, missed that in my PR]